lib/content-services/src/lib/search/components/search-header/search-header.component.html

```diff
@@ -0,0 +1,37 @@
+<div *ngIf="isFilterServiceActive">
+    <div *ngIf="!!category" class="adf-filter">
+        <button mat-icon-button [matMenuTriggerFor]="filter"
+            id="filter-menu-button"
+            #menuTrigger="matMenuTrigger"
+            (click)="onMenuButtonClick($event)"
+            class="adf-filter-button"
+            matTooltip="{{ 'SEARCH.SEARCH_HEADER.FILTER_BY' | translate: { category: col.title || 'Type' } }}">
+            <adf-icon value="adf:filter"
+                [ngClass]="{ 'adf-icon-active': isActive || menuTrigger.menuOpen }"
+                matBadge="&#8288;"
```

Comment:
this color has to come from the scss and has not to be a fixed color

Reply:
Currently the `matBadge="&#8288;"` is a little workaround to have an empty matBadge, actually, the color is defined just below with `matBadgeColor="warn"`